com.regnosys.rosetta/src/com/regnosys/rosetta/Rosetta.xtext

```diff
@@ -765,19 +773,18 @@ RosettaBlueprint:
 ;
 
 BlueprintNodeExp:
-	node=BlueprintNode (reference = RosettaDocReference)? ('then' next=BlueprintNodeExp)? 
+	node=BlueprintNode (reference=RosettaDocReference)?
+	('then' next=BlueprintNodeExp)?
+	(=>'as' identifier=STRING)?
```

Comment:
Cleanup 5: moved `as` up one level in the AST. Makes it easier to format.

Reply:
This actually makes the `as` a lot more useful in DRR. Even though we want to eventually deprecate and remove it.